google/resource_endpoints_service.go

```diff
@@ -0,0 +1,297 @@
+package google
+
+import (
+	"encoding/base64"
+	"encoding/json"
+	"errors"
+	"github.com/hashicorp/terraform/helper/schema"
+	"google.golang.org/api/servicemanagement/v1"
+)
+
+func resourceEndpointsService() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceEndpointsServiceCreate,
+		Read:   resourceEndpointsServiceRead,
+		Delete: resourceEndpointsServiceDelete,
+		Update: resourceEndpointsServiceUpdate,
+		Schema: map[string]*schema.Schema{
+			"service_name": &schema.Schema{
+				Type:     schema.TypeString,
+				Required: true,
+				ForceNew: true,
+			},
+			"openapi_config": &schema.Schema{
+				Type:     schema.TypeString,
+				Optional: true,
+			},
+			"grpc_config": &schema.Schema{
```

Comment:
If the content of the config file changes, does it create a diff? If not, it should

Reply:
It does, yep.